docker/generic/start_proxy.py

```diff
@@ -806,6 +806,10 @@ def gen_proxy_config(args):
         proxy_conf.extend(["--service_account_key", args.service_account_key])
     if args.non_gcp:
         proxy_conf.append("--non_gcp")
+
+    if args.enable_debug:
+        proxy_conf.append("--suppress_envoy_headers=false")
```

Comment:
I am wondering if we should expose this flag here.   Maybe some users may find it useful.

Reply:
@JLXIA PTAL, should we expose this flag to the user?

Reply:
Well, maybe not now. we can do that if there are users required it.